Cython/Build/Dependencies.py

```diff
@@ -534,7 +535,7 @@ def included_files(self, filename):
         for include in self.parse_dependencies(filename)[1]:
             include_path = join_path(os.path.dirname(filename), include)
             if not path_exists(include_path):
-                include_path = self.context.find_include_file(include, None)
+                include_path = self.context.find_include_file(include, (FileSourceDescriptor(filename),))
```

Comment:
Hmm… I'd rather not create a fake code position tuple here. I'll see what I can do about it.

Reply:
I boldly pushed a change to the master branch that makes this unnecessary.